.quickstart/quickstart.yml

```diff
@@ -5,31 +5,22 @@ dbt_versions: ">=1.3.0 <2.0.0"
 
 table_variables:
   salesforce__user_role_enabled:
-    - user_role
+    - UserRole
   salesforce__lead_enabled:
-    - lead
+    - Lead
   salesforce__event_enabled:
-    - event
+    - Event
   salesforce__task_enabled:
-    - task
+    - Task
   salesforce__opportunity_line_item_enabled:
-    - opportunity_line_item
+    - OpportunityLineItem
   salesforce__order_enabled:
-    - order
+    - Order
   salesforce__product_2_enabled:
-    - product_2
+    - Product2
 
 destination_configurations:
   databricks:
     dispatch:
       - macro_namespace: dbt_utils
-        search_order: [ 'spark_utils', 'dbt_utils' ]
-public_models: [
-  "salesforce__contact_enhanced",
-  "salesforce__daily_activity",
-  "salesforce__manager_performance",
-  "salesforce__opportunity_enhanced",
-  "salesforce__opportunity_line_item_enhanced",
-  "salesforce__owner_performance",
-  "salesforce__sales_snapshot"
-]
```

Comment:
Why were these removed?! We need to ensure the public models are included otherwise they will not be visible in the UI.

Reply:
@fivetran-joemarkiewicz This stumped me for a second. For some reason on my git fetch/git pull it did not pick up this `quickstart.yml` update. 

[Is it because this change was merged in September](https://github.com/fivetran/dbt_salesforce/pull/57/files#diff-f19d01eeaef2ed428a11a9bf3e1c4d756c1cca4037a5eae0211961e503851cf3), but [the last release was in July](https://github.com/fivetran/dbt_salesforce/releases/tag/v1.1.1), so it picked up the release but not these updates? 
 
 In any case I've brought these back in.